charts/incubator/gravity/templates/common.yaml

```diff
@@ -0,0 +1,11 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "tc.v1.common.loader.init" . }}
+
+{{/* Render secrets for gravity */}}
+{{- $secrets := include "gravity.secrets" . | fromYaml -}}
+{{- if $secrets -}}
+  {{- $_ := set .Values.secret "secrets" $secrets -}}
```

Comment:
Either this has to be "gravity-secrets"
or on the _secrets.tpl it must be "%s-secrets"

Otherwise it won't match